torch/distributed/benchmarks/benchmark_ddp_rpc.py

```diff
@@ -101,7 +101,7 @@ def _print_cont(msg):
 
 
 def _run_printable(cmd):
-    proc = subprocess.run(shlex.split(cmd), capture_output=True)  # type: ignore[call-overload]
+    proc = subprocess.run(shlex.split(cmd), capture_output=True, check=True)  # type: ignore[call-overload]
```

Comment:
Err, `proc.return_code` is checked literally on the next line
```suggestion
    proc = subprocess.run(shlex.split(cmd), capture_output=True, check=False)  # type: ignore[call-overload]
```

Alternatively, should one replace it with `subprocess.check_output`?